conf/app.ini

```diff
@@ -105,6 +105,10 @@ SUBJECT = %(APP_NAME)s
 HOST =
 ; Do not verify the certificate of the server. Only use this for self-signed certificates
 SKIP_VERIFY = 
+; Use client certificate
+; USE_CERTIFICATE = true
+; CERT_FILE = custom/mailer/cert.pem
+; KEY_FILE = custom/mailer/key.pem
```

Comment:
There are no other options in this file that are commented out. Maybe uncomment them and give them good defaults?

Also, wouldn't it make sense to interpret an empty cert_file / key_file as not using certificates and when a path is provided using certificates?

E.g. 

    ; Provide paths below to use client certificate for smtp connection. 
    CERT_FILE = 
    KEY_FILE =